jjb/dynamic/uperf.yml

```diff
@@ -0,0 +1,101 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |+
+        set -o pipefail
+        set -eux
+        # Disable logging
+        set +x
+        oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL}
```

Comment:
This needs to support skipping the tls verification similar to https://github.com/openshift-scale/scale-ci-pipeline/blob/master/jjb/dynamic/scale-ci_nodevertical.yml#L11.